enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java

```diff
@@ -312,7 +312,7 @@ public Operation claimOperation(String operationId, String language, String user
      * @throws PowerAuthClientException In case communication with PowerAuth Server fails.
      * @throws MobileTokenException When the operation is in incorrect state.
      */
-    private OperationDetailResponse getOperationDetailInternal(String operationId) throws PowerAuthClientException, MobileTokenException {
+    private OperationDetailResponse getOperationDetailInternal(@NotNull String operationId) throws PowerAuthClientException, MobileTokenException {
```

Comment:
🤔 If we are already refactoring the method names, how about renaming this to something like `fetchOpertionDetailFromPowerAuth`?

Reply:
Yes, I will rename the `getOperationDetail` methods.